src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 2L),
+                GT_Values.NI,
                 GT_Values.NI,
                 null,
                 420,
                 120);
+        // Mica
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Zeolite, 41L),
-                GT_Utility.getIntegratedCircuit(1),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Mica, 38L),
+                ItemList.Cell_Empty.get(11L),
                 GT_Values.NF,
-                Materials.Oxygen.getGas(18000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Calcium, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 27L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 9L),
+                Materials.Fluorine.getGas(4000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 6L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 11L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                944,
+                260,
                 120);
+        // Alunite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
-                GT_Utility.getIntegratedCircuit(1),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Alunite, 52L),
+                ItemList.Cell_Empty.get(11L),
                 GT_Values.NF,
-                Materials.Oxygen.getGas(3000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 2L),
-                GT_Values.NI,
-                GT_Values.NI,
-                GT_Values.NI,
+                Materials.Hydrogen.getGas(12000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 11L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                100,
-                480);
+                210,
+                120);
+        // Zeolite
```

Comment:
this should of course be 141 dust if the water is voided or not. but since the electrolyzer cant handle that, it is tricky.

Maybe divide by 3 and do: 47 dust -> 3 tiny Na (or chance based) + 12 tiny Ca (or chance based) + 3Al +27 SiO2

Reply:
I dont see a good solution though. so do what you think is best.

Reply:
Zeolites are a group of minerals, without a clear formula. Meybe we can just change it into something... normal?

Reply:
fixed by changing material composition is materials.java